testdata/helm/memcached-operator/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -12,9 +12,10 @@ spec:
       - name: kube-rbac-proxy
         securityContext:
           allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-              - ALL
```

Comment:
The samples we need to uncomment

Why? 
- So that we can show how it works for the users
- We can ensure these requirements against k8s with ansible/helm and indeed go
- Also, when we test it against OCP versions when we downstream it we will be sure that can works there

Reply:
To help I did it here: https://github.com/operator-framework/operator-sdk/pull/5872/files